github/orgs_audit_log_test.go

```diff
@@ -421,3 +436,128 @@ func TestAuditEntry_Marshal(t *testing.T) {
 
 	testJSONMarshal(t, u, want)
 }
+
+func TestAuditEntry_Unmarshal(t *testing.T) {
```

Comment:
I added this test because the pipeline was breaking/"complaining" about the percentage of test coverage.